test/screens/play/play_page_test.dart

```diff
@@ -75,4 +76,24 @@ void main() {
     final specialCard = tester.widget<Card>(cardFinder);
     expect(specialCard.color, specialCardColour);
   });
+
+  testWidgets('should display correct count in deck and discard piles', (WidgetTester tester) async {
+    //ugly hack to set a stupid size, so this test doesn't complain about random overflows
+    tester.binding.window.physicalSizeTestValue = Size(2000, 1920);
```

Comment:
We can remove this (and the other one in here) with the `ListView` fix :grinning: